crypto/stark/assets/Trace.sol

```diff
@@ -0,0 +1,40 @@
+
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+import '../interfaces/ConstraintInterface.sol';
+import '../default_cs.sol';
+
+abstract contract {name}Trace is DefaultConstraintSystem({constraint_degree}, {num_rows}, {num_cols}, {blowup}) \{
+    // This lets us map rows -> inverse index,
+    // In complex systems use a autogen binary search.
+    function row_to_offset(uint256 row) internal pure override returns(uint256) \{
```

Comment:
This is an unused function which we should probably remove. Have been meaning to make a pr to do it, maybe here is best.